src/styles/components/raise-work-order-link.scss

```diff
@@ -0,0 +1,56 @@
+.raiseWorkOrderLink {
+  text-decoration: none;
+  margin-bottom: 10px;
+  display: block;
+
+  outline: none !important;
+
+  &-container {
+    border: 2px solid hsl(180, 2%, 92%);
+    background: hsl(180, 2%, 97%);
+    border-radius: 5px;
+    padding: 10px 20px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &-title {
+    color: hsl(180, 2%, 10%);
```

Comment:
HSL? I never saw anyone go with that over rgb or #HEXCODE.
I'm quite curious - what influenced the choice?